sql/log.cc

```diff
@@ -10736,4 +10755,51 @@ void wsrep_register_binlog_handler(THD *thd, bool trx)
   DBUG_VOID_RETURN;
 }
 
+void wsrep_append_xid_string(String &xid_string, const XID* xid)
+{
+  DBUG_ASSERT(xid && !xid->is_null());
+  xid_string.reserve(xid->gtrid_length + xid->bqual_length + 9);
```

Comment:
Where that 9 comes from and does this take a account different character sets (i.e. multi-byte character sets) ?